### DISCUSSION TITLE
CRSF Token missing

### DISCUSSION THREAD

**Discussion by Karthik-PlantSense** [apache/airflow#18832] [Q&A]
i am trying to create a plugin which support rest-api whenever i try to call the api
on POST method it is saying csrf token and also i am using the `@csrf.exempt`
function also i got the same error

**Apache Airflow version**: 2.0.0, LocalExecutor

**Environment**: linux centos7

**Airflow Configuration**

`rbac=True`

please someone help me on this one...
@imbaczek

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by mik-laj** [comment]
What version of Airflow are you using?Doo you use RBAC UI or not?

**Comment by Karthik-PlantSense** [comment]
@mik-laj 
i'm using
`airflow version 2.0` 
`rbac=True`
